### PR TITLE
Update validation message.

### PR DESCRIPTION
**Pull request by foxish**
Fixes validation message when patching a StatefulSet

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37653)

**Comment by foxish** [comment]
ping. Not high priority but we should probably fix this message as it is inaccurate.

File: pkg/apis/apps/validation/validation.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func ValidateStatefulSetUpdate(statefulSet, oldStatefulSet *apps.StatefulSet) fi
 	statefulSet.Spec.Template.Spec.Containers = oldStatefulSet.Spec.Template.Spec.Containers
 
 	if !reflect.DeepEqual(statefulSet.Spec, oldStatefulSet.Spec) {
-		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec"), "updates to statefulset spec for fields other than 'replicas' are forbidden."))
+		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec"), "updates to statefulset spec for fields other than 'replicas' and 'containers' is forbidden."))
```
Comment (enisoc) [inline pkg/apis/apps/validation/validation.go]: I think it should still be "are forbidden". The subject is "updates" which is plural.

**Comment by enisoc** [comment]
LGTM

**Comment by foxish** [comment]
@k8s-bot verify test this

**Comment by foxish** [comment]
@k8s-bot verify test this

**Comment by foxish** [comment]
@k8s-bot test this

**Comment by rmmh** [comment]
@k8s-bot test this

**Pull request merged**